ci-operator/config/open-cluster-management/search-collector/open-cluster-management-search-collector-main.yaml

```diff
@@ -0,0 +1,21 @@
+build_root:
+  image_stream_tag:
+    name: builder
+    namespace: open-cluster-management
+    tag: go1.16-linux-amd64
+images:
+- dockerfile_path: Dockerfile
+  from: base
```

Comment:
Here you're telling the images builder to use the `base` image, but you don't have a `base_images` stanza.

Need to add this section:
```
base_images:
  base:
    name: ubi-minimal
    namespace: ocp
    tag: "8"
```

I also just noticed that this section was deleted from the onboarding docs. I'll get it added back.

Reply:
Thanks for pointing this out!